app/components/content/results_box_component.rb

```diff
@@ -0,0 +1,25 @@
+module Content
+  class ResultsBoxComponent < ViewComponent::Base
+    attr_reader :title, :heading, :fee, :course_length, :funding, :text, :link_text, :link_target, :border_color, :show_title
+
+    include ContentHelper
+
+    def initialize(heading:, fee:, course_length:, funding:, text:, link_text:, link_target:, title: nil, border_color: :grey)
+      super
+
+      @title = substitute_values(title)
+      @heading = substitute_values(heading)
+      @fee = substitute_values(fee)
+      @course_length = substitute_values(course_length)
+      @funding = substitute_values(funding)
+      @text = substitute_values(text)
+      @link_text = link_text
+      @link_target = link_target
+      @border_color = border_color
+    end
+
+    def border_class
```

Comment:
I think we could get rid of this method as you could just interpolate the border color onto the class in the html